content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -144,7 +144,7 @@ EOF
 # Get commands with basic output
 kubectl get services                          # List all services in the namespace
 kubectl get pods --all-namespaces             # List all pods in all namespaces
-kubectl get pods -o wide                      # List all pods in the namespace, with more details
+kubectl get pods -o wide --all-namespaces     # List all pods in the namespace, with more details
```

Comment:
@prasadph The revised command does not match the description.

Maybe this would be more clear:
```suggestion
kubectl get pods -o wide                      # List all pods in the current namespace, with more details
```
?